modules/evm/src/runner/stack.rs

```diff
@@ -772,7 +772,7 @@ impl<'vicinity, 'config, T: Config> StackStateT<'config> for SubstrateStackState
 	}
 
 	fn reset_storage(&mut self, address: H160) {
-		let _ = <AccountStorages<T>>::clear_prefix(address, u32::MAX, None);
```

Comment:
new bencher will count limit as written operation so I'm changing this to drain_prefix to avoid counting any write operation

Reply:
best to leave a comment explain this for future readers